src/compositions/useCustomDefaultFlowRunsFilter.ts

```diff
@@ -1,8 +1,27 @@
 import { useLocalStorage } from '@prefecthq/vue-compositions'
-import { getCacheKey, FlowRunsFilter } from '..'
+import { ComputedRef, computed, Ref } from 'vue'
+import { getCacheKey, FlowRunsFilter, SavedSearchFilter, mapper } from '..'
 
 const customDefaultFlowRunsFilterKey = getCacheKey('prefect-ui-library-custom-default-flow-runs-filter')
 
-export function useCustomDefaultFlowRunsFilter(): ReturnType<typeof useLocalStorage<FlowRunsFilter | null>> {
-  return useLocalStorage<FlowRunsFilter>(customDefaultFlowRunsFilterKey)
+type UseCustomDefaultFlowRunsFilter = {
+  value: Ref<SavedSearchFilter | null>,
+  set: (value: SavedSearchFilter) => void,
+  remove: () => void,
+  asFlowRunsFilter: ComputedRef<FlowRunsFilter | null>,
+}
+
+const { value, set } = useLocalStorage<SavedSearchFilter>(customDefaultFlowRunsFilterKey)
+
+export function useCustomDefaultFlowRunsFilter(): UseCustomDefaultFlowRunsFilter {
```

Comment:
I'm not 100% sure but it looks like where you're using this you check if this has a null value and then you coalesce the system default. Could this just automatically return the system default if the local storage value is null? It could also return an `isCustom` boolean so implementations would know.